ff/src/fields/models/quadratic_extension.rs

```diff
@@ -192,11 +192,17 @@ impl<P: QuadExtParameters> One for QuadExtField<P> {
 
 impl<P: QuadExtParameters> Field for QuadExtField<P> {
     type BasePrimeField = P::BasePrimeField;
+    type BasePrimeFieldArr = Vec<Self::BasePrimeField>;
```

Comment:
I'm not sure if we could use another container (an array?) given that the fields are being buit using macros and so we might be able to postpone this assignment till the extension degree over the prime field is known.